packages/riipen-ui/src/components/Accordion.jsx

```diff
@@ -48,17 +49,35 @@ const Accordion = props => {
       )}
       {...other}
     >
-      <AccordionContext.Provider value={contextValue}>
-        {summary}
-      </AccordionContext.Provider>
-      <Collapse in={expanded}>
-        <div
-          aria-labelledby={summary.props.id}
-          id={summary.props["aria-controls"]}
-        >
-          {children}
-        </div>
-      </Collapse>
+      {provideContextToChildren && (
```

Comment:
im not super in on the Context stuff, what would be the downside of always wrapping the children with the context?